LayoutTests/fast/dom/document-contentType-DOMParser.html

```diff
@@ -80,6 +80,9 @@
 shouldThrow('new DOMParser().parseFromString(xmlContent, "text/XML").contentType', "'TypeError: Type error'");
 shouldThrow('new DOMParser().parseFromString(htmlContent, "TEXT/html").contentType', "'TypeError: Type error'");
 
+var htmlContentWithSpace = "\n        <div class=\"bpx-player-container\"></div>";
```

Comment:
Please add a similar test case for setting element.innerHTML to validate that it doesn't trim whitespace.